api/src/services/export/message-mapper.js

```diff
@@ -44,14 +44,18 @@ const buildHistory = task => {
   return history;
 };
 
-const formatDate = date => {
+const formatDate = (date, filters) => {
   if (!date) {
     return '';
   }
-  return moment(date).valueOf();
+  if (filters && filters.human && filters.human === 'true'){
+    return moment(date).toISOString();
+  } else {
+    return moment(date).valueOf();
+  }
```

Comment:
Let's fix this duplicate while we're here. Consider making a new date formatting module which can be called from all the mappers, eg:

```js
const dateFormat = require('./date-format');
...
const humanReadable = filters && filters.human && filters.human === 'true';
dateFormat.format(date, { humanReadable });
```